2022q3/kde.adoc

```diff
@@ -0,0 +1,41 @@
+=== KDE on FreeBSD
+
+Links: +
+link:https://freebsd.kde.org/[KDE FreeBSD] URL: link:https://freebsd.kde.org/[https://freebsd.kde.org/] +
+link:https://community.kde.org/FreeBSD[KDE Community FreeBSD] URL: link:https://community.kde.org/FreeBSD[https://community.kde.org/FreeBSD]
+
+Contact: Adriaan de Groot <kde@FreeBSD.org>
+
+The KDE on FreeBSD project packages the software from the KDE Community, along with dependencies and related software, for the FreeBSD ports tree.
+The software includes a full desktop environment called KDE Plasma (for both X11 and Wayland) and hundreds of applications that can be used on any FreeBSD machine.
+
+The KDE team (kde@) is part of desktop@ and x11@, building the software stack to make FreeBSD beautiful and usable as a daily-driver graphics-based desktop machine.
+The notes below describe *mostly* ports for KDE, but also include items that are important for the entire desktop stack.
+
+==== Qt6 Landed
+
+The big news in the KDE ports is not directly KDE-related. Qt6 has landed, which prepares us for the next generation of Qt-based applications.
+It is now possible to have `USES=qt:6` to select the new Qt version. Some ports have been flavorized to make use of the new version.
+KDE itself is not affected: the upstream work on KDE Frameworks for Qt6 is not yet completed. Most of the KDE Frameworks
+will compile with Qt6, but that is not important for FreeBSD ports yet. With
+package:devel/qt6[] you get Qt 6.4.0, released at the end of the quarter.
+
+==== KDE Stack
+
+KDE Gear releases happen each quarter, KDE Plasma updates once a month, and KDE Frameworks have a new release each month as well.
+These (large) updates land shortly after their upstream release and are not listed separately.
+
+* KDE Frameworks 5 is now at version 5.98 (latest monthly release from September 2022).
+* KDE Gear is now version 22.08.1 (update for September 2022).
+* KDE Plasma is now version 5.24.6 (update for July 2022).
+
+Note that KDE Plasma 5.25 has been released upstream, but is waiting on fixes elsewhere in FreeBSD before it can land in the ports tree.
```

Comment:
Can you elaborate on "elsewhere"? Do you mean ports it depends on, base FreeBSD, both?

Reply:
Not a direct answer, but I'm aware of KDE bug [458356 – Plasma 5.25.x not launching on FreeBSD](https://bugs.kde.org/show_bug.cgi?id=458356).

Reply:
Going to wait on the author answer, as this could be either as far as I can tell. Or it might be hard or impossible to tell.

Reply:
I was thinking "there's some PAM issue holding us back" but don't have a specific example there, Graham's upstream bug is a good one though that needs examination.